### ISSUE TITLE
Error: Column start (18) greater than line length (17)

### ISSUE THREAD

**Issue by E-SHADOW** [AtomLinter/linter-stylelint#107]
Hello :) 
I need help please ..
![2016-03-02_22-40-10](https://cloud.githubusercontent.com/assets/15945528/13474799/4ccdd48a-e0c8-11e5-82d5-6207a9e19d9d.jpg)

And thank you :)

**Comment by Arcanemagus** [comment]
Are you able to share the file that triggered this?

**Comment by E-SHADOW** [comment]
You mean the file in the path ?

**Comment by Arcanemagus** [comment]
No, the file that you were editing that triggers this error.

**Comment by Arcanemagus** [comment]
@E-SHADOW are you able to share that file? Without the file it's impossible to determine where this bug is triggered from and where it needs to be fixed.

**Comment by ilikescience** [comment]
@Arcanemagus I'm getting the same error. It seems to pop up whenever I add the "*"s into the paths here:

```
// import any vendor files
@import "vendor/*";

// project-wide styles - styles single elements
@import "global/*";

// component styles - styles groups of elements
@import "components/*";

// page layouts - styles groups of components
@import "layouts/*";
```

Could be a problem where that `/*` is registering as the start of a comment?

**Comment by Arcanemagus** [comment]
Ah ha, managed to trigger that when using SCSS syntax on that example, looking into this.

**Comment by Arcanemagus** [comment]
@ilikescience your issue should be fixed once the changes in https://github.com/stylelint/stylelint/pull/909 are released. Leaving this open as @E-SHADOW reported this before that rule existed so they must be running into this from a different cause.

**Comment by Arcanemagus** [comment]
@ilikescience coincidentally `stylelint` pushed a patch release that fixed that, pushed out v2.0.1 of this plugin that integrates that.

**Comment by ilikescience** [comment]
@Arcanemagus thanks!

**Comment by Arcanemagus** [comment]
Marking this as closed due to a lack of response from @E-SHADOW. If you are still seeing this please re-open this issue, or file a new one. Thanks!
